python/minidaqapp/nanorc/readout_gen.py

```diff
@@ -297,33 +288,63 @@ def generate(NETWORK_ENDPOINTS,
+                            latency_buffer_size = LATENCY_BUFFER_SIZE,
+                            region_id = 0,
+                            element_id = idx,
+                        ),
+                        rawdataprocessorconf= rconf.RawDataProcessorConf(
+                            region_id = 0,
+                            element_id = idx,
+                            enable_software_tpg = SOFTWARE_TPG_ENABLED,
+                        ),
+                        requesthandlerconf= rconf.RequestHandlerConf(
+                            latency_buffer_size = LATENCY_BUFFER_SIZE,
+                            pop_limit_pct = 0.8,
+                            pop_size_pct = 0.1,
+                            region_id = 0,
+                            element_id = idx,
+                            output_file = f"output_{idx + MIN_LINK}.out",
+                            stream_buffer_size = 8388608,
+                            enable_raw_recording = RAW_RECORDING_ENABLED,
+                        )
+                        )) for idx in range(MIN_LINK, MAX_LINK)
             ] + [
-                (f"data_recorder_{idx}", dr.Conf(
-                        output_file = f"output_{idx + MIN_LINK}.out",
-                        compression_algorithm = "None",
-                        stream_buffer_size = 8388608)) for idx in range(NUMBER_OF_DATA_PRODUCERS)
+                (f"tp_datahandler_{TOTAL_NUMBER_OF_DATA_PRODUCERS + idx}", rconf.Conf(
+                        readoutmodelconf= rconf.ReadoutModelConf(
+                            source_queue_timeout_ms= QUEUE_POP_WAIT_MS,
+                            # fake_trigger_flag=0, default
+                            region_id = 0,
+                            element_id = TOTAL_NUMBER_OF_DATA_PRODUCERS + idx,
+                        ),
+                        latencybufferconf= rconf.LatencyBufferConf(
+                            latency_buffer_size = LATENCY_BUFFER_SIZE,
+                            region_id = 0,
+                            element_id = TOTAL_NUMBER_OF_DATA_PRODUCERS + idx,
+                        ),
+                        rawdataprocessorconf= rconf.RawDataProcessorConf(
+                            region_id = 0,
+                            element_id = TOTAL_NUMBER_OF_DATA_PRODUCERS + idx,
+                            enable_software_tpg = False,
+                        ),
+                        requesthandlerconf= rconf.RequestHandlerConf(
+                            latency_buffer_size = LATENCY_BUFFER_SIZE,
+                            pop_limit_pct = 0.8,
+                            pop_size_pct = 0.1,
+                            region_id = 0,
+                            element_id = TOTAL_NUMBER_OF_DATA_PRODUCERS + idx,
+                            # output_file = f"output_{idx + MIN_LINK}.out",
+                            # stream_buffer_size = 8388608,
+                            enable_raw_recording = False,
```

Comment:
I leave it set to False and the output_file and stream buffer size commented since I don't see anything like that in the previous version